techmatters/terraso-mobile-client
# fix: hide depth-edit modal behind permission check

## Description

Hide the controls to edit depth intervals on sites behind a role-based permission check.

### Checklist

### Related Issues
#1494

### Verification steps

Open a site's soil screen as a viewer and verify that the vertical "more" button is not visible. Open it as an owner, contributor, or manager and verify that it is still visible and opens the editing modal.
